### PR TITLE
Replaces "NanoTrasen Relation" with "Economic Status"

### PR DESCRIPTION
**Pull request by furrycactus**
As the titles says, changes out the arbitrary "NanoTrasen Relation" with various levels of economic status. It wasn't super clear what the loyalty level effected, and it doesn't make much sense where contractors are concerned either. Here's what it looks like.

https://i.imgur.com/l7T550Z.png

https://i.imgur.com/2J7R9SO.png

### CODE REVIEW

**Comment by furrycactus** [comment]
!review

**Comment by Doxxmedearly** [comment]
Please rename the "nanotrasen relation" and '"loyalty" vars to make more sense.

File: SQL/migrate/V001__Initial_schema.sql [hunk 1]
```diff
@@ -320,7 +320,7 @@ CREATE TABLE `ss13_characters` (
   `citizenship` text,
   `faction` text,
   `religion` text,
-  `nt_relation` text,
+  `eco_stat` text,
```
Comment (NonQueueingMatt) [inline SQL/migrate/V001__Initial_schema.sql:323]: Don't change these.

File: SQL/migrate/V041__SQL-Cleanup.sql [hunk 2]
```diff
@@ -312,8 +312,8 @@ ALTER TABLE `ss13_characters`
 	CHANGE COLUMN `citizenship` `citizenship` MEDIUMTEXT NULL COLLATE 'utf8mb4_unicode_ci' AFTER `home_system`,
 	CHANGE COLUMN `faction` `faction` MEDIUMTEXT NULL COLLATE 'utf8mb4_unicode_ci' AFTER `citizenship`,
 	CHANGE COLUMN `religion` `religion` MEDIUMTEXT NULL COLLATE 'utf8mb4_unicode_ci' AFTER `faction`,
-	CHANGE COLUMN `nt_relation` `nt_relation` MEDIUMTEXT NULL COLLATE 'utf8mb4_unicode_ci' AFTER `religion`,
-	CHANGE COLUMN `uplink_location` `uplink_location` MEDIUMTEXT NULL COLLATE 'utf8mb4_unicode_ci' AFTER `nt_relation`,
+	CHANGE COLUMN `eco_stat` `eco_stat` MEDIUMTEXT NULL COLLATE 'utf8mb4_unicode_ci' AFTER `religion`,
+	CHANGE COLUMN `uplink_location` `uplink_location` MEDIUMTEXT NULL COLLATE 'utf8mb4_unicode_ci' AFTER `eco_stat`,
```
Comment (NonQueueingMatt) [inline SQL/migrate/V041__SQL-Cleanup.sql:316]: Don't change these.

File: code/game/gamemodes/game_mode.dm [hunk 3]
```diff
@@ -459,7 +459,7 @@ var/global/list/additional_antag_types = list()
 		else if(man.mind.assigned_job)
 			var/datum/job/job = man.mind.assigned_job
 			var/evil = 0
-			if(man.client.prefs.nanotrasen_relation == COMPANY_OPPOSED || man.client.prefs.nanotrasen_relation == COMPANY_SKEPTICAL)
+			if(man.client.prefs.economic_status == ECONOMIC_POOR || man.client.prefs.economic_status == ECONOMIC_UNDERPAID)
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/game_mode.dm:462]: The things here don't really make sense if it's economy status, tbh. I'd axe this proc.

**Comment by furrycactus** [comment]
I JUST WANTED TO CHANGE SOMETHING THAT LOOKED EASY

AAAAAAAAAA

**Comment by Doxxmedearly** [comment]
OH NO I LED YOU ASTRAY I'M SORRY T_T

**Comment by NonQueueingMatt** [comment]
reopen when ready

**Pull request closed without merging**